examples/jest/ts/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
maybe this file should go one directory up? is that more typical of how it's used?
then you wouldn't need the `alias` rule in the /BUILD.bazel

Reply:
fixing